examples/sample/release.yaml

```diff
@@ -3,29 +3,33 @@ kind: Release
 metadata:
   name: wordpress-example
 spec:
+#  rollbackLimit: 3
   forProvider:
     chart:
       name: wordpress
       repository: https://charts.bitnami.com/bitnami
       version: 9.3.19
+#      pullSecretRef:
+#        name: museum-creds
+#        namespace: default
     namespace: wordpress
     values: |
       param1: value1
       param2: 1
     set:
       - name: param1
         value: value2
-    valuesFrom:
-      - configMapKeyRef:
-          key: values.yaml
-          name: default-vals
-          namespace: wordpress
-          optional: false
-      - secretKeyRef:
-          key: svalues.yaml
-          name: svals
-          namespace: wordpress
-          optional: false
+#    valuesFrom:
```

Comment:
forgotten to cleanup commented lines?

Reply:
No, this was intentional since this is an example file.